docs/userguide/format_parse/messages/index.md

```diff
@@ -35,6 +35,14 @@ and formatted elements, then translators would not be able to rearrange the
 pieces, and they would have a hard time translating each of the string
 fragments.
 
+*Note: the information on the rest of this page describes how to use the 
+current, long-standing ICU API `MessageFormat`.
+A successor to this API is
+being developed [in a working group](https://github.com/unicode-org/message-format-wg),
+which has created a [draft of a specification](https://github.com/unicode-org/message-format-wg/tree/main/spec)
```

Comment:
Nitpick: created a [draft specification](...) ?
s / a draft of a specification / a draft specification /